src/Symfony/Component/Testing/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "symfony/testing",
+    "type": "library",
+    "description": "Symfony Testing Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
+    },
+    "autoload": {
+        "psr-0": { "Symfony\\Component\\Testing\\": "" }
+    },
+    "autoload-dev": {
+        "files": [ "Resources/bootstrap/symfony.php" ]
+    },
```

Comment:
you are aware that this won't be used when the package is required in another package? Dev config will only be tracked when the package is the root package.